snow/networking/router/chain_router_test.go

```diff
@@ -1603,3 +1625,14 @@ func TestHandleSimplexMessage(t *testing.T) {
 	chainRouter.HandleInbound(context.Background(), inboundMsg)
 	require.True(t, receivedMsg)
 }
+
+func createSubscriberAndChangeNotifier() (*block.ChangeNotifier, common.Subscription) {
+	var cn block.ChangeNotifier
+
+	subscription := func(ctx context.Context) (common.Message, error) {
+		<-ctx.Done()
+		return common.Message(0), ctx.Err()
+	}
+
+	return &cn, subscription
+}
```

Comment:
Why do we have `createSubscriberAndChangeNotifier`? The returned `subscription` doesn't have any state related to the `ChangeNotifier` (actually any state at all).

It seems like the callers of this should just define `var cn block.ChangeNotifier` themselves and reference a single static `noopSubscription` function.
```suggestion
func noopSubscription(ctx context.Context) (common.Message, error)
	<-ctx.Done()
	return common.Message(0), ctx.Err()
}
```